### PR TITLE
adding parameters to server and updating e2e

### PR DESCRIPTION
**Pull request by OmerBenHayun**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,7 +37,8 @@ jobs:
     - make install-tools
     before_script:
     - make etcd
-    - make server
+    - make north-server
+    - make south-server
```
Comment (roytman) [inline .travis.yml:41]: I don't think for CI (travis) we need both NB and SB deployments. One of them should be enough

File: pkg/cmd/server/server.go [hunk 2]
```diff
@@ -24,17 +26,25 @@ const UNIX_SOCKET = "/tmp/ovsdb-etcd.sock"
...
+	schemaFile         = flag.String("schema-file", "", "schema-file")
+	schemaName         = flag.String("schema-name", "", "schema-name")
+	loadServerDataFlag = flag.Bool("load-server-data", false, "load-server-data")
 )
 
 func main() {
 	klog.InitFlags(nil)
 	flag.Parse()
+
+	fmt.Println("start the ovsdb-etcd server with the following arguments:")
```
Comment (roytman) [inline pkg/cmd/server/server.go:44]: It is good for development, and should be removed later. 
Please add the // TODO comment.

File: .travis.yml (hunk 1 above)
Comment (OmerBenHayun) [reply .travis.yml]: Thanks,Done

File: pkg/cmd/server/server.go (hunk 2 above)
Comment (OmerBenHayun) [reply pkg/cmd/server/server.go:45]: Thanks,Done

**Pull request merged**